heartexlabs/label-studio
# docs: Fix admonition rendering

Fix https://github.com/heartexlabs/label-studio/issues/4538

### How to QA

These should all work now:

https://deploy-preview-4541--label-studio-docs-new-theme.netlify.app/guide/install.html#Install-with-Docker-on-nix
https://deploy-preview-4541--label-studio-docs-new-theme.netlify.app/guide/signup.html#Create-an-account
